otcextensions/osclient/cce/v2/node_pool.py

```diff
@@ -0,0 +1,446 @@
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may
+#   not use this file except in compliance with the License. You may obtain
+#   a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+#
+'''CCE Cluster Nodes v2 action implementations'''
```

Comment:
is it `node actions` or `node pool actions`?

Reply:
copy and paste ^^: issue is solved.